toolkit/docs/building/building.md

```diff
@@ -487,6 +490,20 @@ sudo make hydrate-rpms PACKAGE_ARCHIVE=./rpms.tar.gz
 
 > Build the go tools from source as needed.
 
+#### `REFRESH_WORKER_CHROOT=...`
+
+##### `REFRESH_WORKER_CHROOT=`**`n`**
+
+> If exists, don't attempt to rebuild the worker chroot, even if the its build script, its manifest, or the packages it consists of have been modified in the local repository.
+
+##### `REFRESH_WORKER_CHROOT=`**`y`** *(default)*
+
+> Rebuild the worker chroot every time at least one of the following has changes:
```

Comment:
```suggestion
> Rebuild the worker chroot every time if at least one of the following has changes:
```

Reply:
I believe this is correct, but I've noticed a typo in "changes" - I wanted to say "changed". Thanks!